routes/teams.js

```diff
@@ -40,4 +43,115 @@ router.get('/team', function(req, res) {
     });
 });
 
+
+// when players endpoint is hit, call the API/DB using that team_id
+router.get('/players', function(req, res) {
+  team_id = req.query["team_id"];
+  console.log("team id is: "+team_id);
+  players = fetchPlayers(team_id);
```

Comment:
You are setting players to the fetchplayes method here, but fetchplayers does not return anything